src/Tasks/Microsoft.NET.Build.Tasks/sdk/Sdk.props

```diff
@@ -30,54 +30,8 @@ Copyright (c) .NET Foundation. All rights reserved.
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(ImportDirectoryBuildProps)' == 'true' and '$(DirectoryBuildPropsPath)' == ''">
-    <_DirectoryBuildPropsFile Condition="'$(_DirectoryBuildPropsFile)' == ''">Directory.Build.props</_DirectoryBuildPropsFile>
-    <_DirectoryBuildPropsBasePath Condition="'$(_DirectoryBuildPropsBasePath)' == ''">$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), '$(_DirectoryBuildPropsFile)'))</_DirectoryBuildPropsBasePath>
-    <DirectoryBuildPropsPath Condition="'$(_DirectoryBuildPropsBasePath)' != '' and '$(_DirectoryBuildPropsFile)' != ''">$([System.IO.Path]::Combine('$(_DirectoryBuildPropsBasePath)', '$(_DirectoryBuildPropsFile)'))</DirectoryBuildPropsPath>
-  </PropertyGroup>
-
-  <Import Project="$(DirectoryBuildPropsPath)" Condition="'$(ImportDirectoryBuildProps)' == 'true' and exists('$(DirectoryBuildPropsPath)')"/>
-
-  <PropertyGroup>
-    <ImportDirectoryBuildProps>false</ImportDirectoryBuildProps>
-  </PropertyGroup>
-
-  <!-- If ArtifactsPath or UseArtifactsOutput are set, then import .props to set ArtifactsPath here, so that BaseIntermediateOutputPath can be
-       set in the ArtifactsPath.
-       If the .props file is not imported here, it will be imported from Microsoft.NET.DefaultOutputPaths.targets, so that artifacts output
-       properties can be set directly in the project file too (only in that case they won't affect the intermediate output). -->
-  <Import Project="$(MSBuildThisFileDirectory)..\targets\Microsoft.NET.DefaultArtifactsPath.props"
-          Condition="'$(UseArtifactsOutput)' == 'true' Or '$(ArtifactsPath)' != ''"/>
-
-  <PropertyGroup Condition="'$(UseArtifactsOutput)' == 'true'">
-    <UseArtifactsIntermediateOutput Condition="'$(UseArtifactsIntermediateOutput)' == ''">true</UseArtifactsIntermediateOutput>
-    <ArtifactsProjectName Condition="'$(ArtifactsProjectName)' == ''">$(MSBuildProjectName)</ArtifactsProjectName>
-  </PropertyGroup>
-  
-  <PropertyGroup Condition="'$(UseArtifactsOutput)' == 'true' And '$(BaseIntermediateOutputPath)' == '' And '$(UseArtifactsIntermediateOutput)' == 'true'">
-    <BaseIntermediateOutputPath Condition="'$(IncludeProjectNameInArtifactsPaths)' == 'true'">$(ArtifactsPath)\obj\$(ArtifactsProjectName)\</BaseIntermediateOutputPath>
-    <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)' == ''">$(ArtifactsPath)\obj\</BaseIntermediateOutputPath>
-  </PropertyGroup>
 
-  <!-- Record whether ArtifactsPath / UseArtifactsOutput was set at this point in evaluation.  We will generate an error if these properties are set
-       after this point (ie in the project file). -->
-  <PropertyGroup Condition="'$(UseArtifactsOutput)' == 'true'">
-    <_ArtifactsPathSetEarly>true</_ArtifactsPathSetEarly>
+    <CustomAfterDirectoryBuildProps>$(MSBuildThisFileDirectory)UseArtifactsOutputPath.props</CustomAfterDirectoryBuildProps>
```

Comment:
Do we need to defensively check if `CustomAfterDirectoryBuildProps` is empty here? Since this is used in an `Import` we can't allow multiple semicolon-delimited values, but we also can't just accept any arbitrary user input here. cc @rainersigwald / @dsplaisted - this means that no one else can ever use this property for SDK projects, right?

Reply:
I happily retract the statement I made in triage: you can't Import from an Item, but it appears you can import from a semicolon-delimited Property:
https://github.com/dotnet/msbuild/pull/2014

I'll change this to just add it to the end of the list.

Reply:
> <Import Project="import1.props;import2.props" Condition="Exists('import1.props;import2.props')" />

I assume you should be prudent to not add any paths here that don't exist or because the import will then probably get skipped for all paths.

Reply:
Definitely agree